.gitignore

```diff
@@ -0,0 +1,2 @@
+/target
+.DS_Store
```

Comment:
Maybe you can use this link: https://github.com/github/gitignore/blob/main/Rust.gitignore

That repo also contains a bunch of useful .gitignores!